modules/distribution/scripts/wso2server.sh

```diff
@@ -304,6 +304,7 @@ do
     -Djava.net.preferIPv4Stack=true \
     -Dcom.ibm.cacheLocalHost=true \
     -DworkerNode=false \
+    -Dtenant.idle.time=153722867280912 \
```

Comment:
@dilini-muthumala 
The expectation for the above fix was to avoid tenant unloading issue which can cause issues such as execution plan not runnig properly, etc. Hence we increased the tenant unloading time by above amount which is Long.MAX_VALUE/60000.